docs/databases/uploads.md

```diff
@@ -69,6 +69,40 @@ If you have a file larger than 200 MB, the workaround here is to:
 2. Upload those files one by one. Metabase will create a new model for each sheet.
 3. Consolidate that data by creating a new question or model that joins the data from those constituent models created by each upload.
 
+## Date formats
+
+For now, Metabase only recognizes dates and datetimes from strings in uploaded CSVs with the following formats:
+
+### Dates
+
+Represents the year, month, and day without time information.
+
+**Format**: `yyyy-MM-dd`
+
+**Example**: `2023-01-01`
+
+### Datetimes
+
+Represents the year, month, day, hour, minute, second, and millisecond with optional fractional seconds.
```

Comment:
Minute and second are also optional.

e.g. `2023-01-01 00` is valid

Also millisecond is a type of fractional second.

`2023-01-01 00:00:00.0` is valid, as is `2023-01-01 00:00:00.0000000`, although that precision will likely be rounded by the database.